### PR TITLE
feat(Pagination): align with DS 5.7

### PR DESCRIPTION
**Pull request by plagoa**
- Included a commit for the `BaseDropdown` styles here as it's used in the `Pagination` component. It should help with other stories

### CODE REVIEW

File: packages/core/src/components/Pagination/Pagination.styles.tsx
[the remark this answers is not in the document]
Comment (plagoa) [reply packages/core/src/components/Pagination/Pagination.styles.tsx:1]: Right, this also changes DS3. I'll update to keep current DS3 behavior.

**Review by zettca — APPROVED** [review]
👌

File: packages/core/src/components/Pagination/Pagination.styles.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const { staticClasses, useClasses } = createClasses("HvPagination", {
     flexWrap: "wrap",
     marginTop: theme.space.sm,
     [`& $pageSizeInput`]: {
-      ...(theme.typography.label as CSSProperties),
+      ...(theme.typography.caption1 as CSSProperties),
```
Comment (MEsteves22) [inline packages/core/src/components/Pagination/Pagination.styles.tsx:21]: In DS5 it should be `caption2` and not `caption1` right?

**Review by MEsteves22 — COMMENTED** [review]
I also noticed the following:

- I don't think it's not on Figma, but wouldn't it be better to also change the size of the options text:

Screenshot 2023-10-24 at 15 57 21

- Also, I noticed the background color overflows the input when focused:

Screenshot 2023-10-24 at 15 57 35

**Review by MEsteves22 — APPROVED** [review]

**Pull request merged**